restapi/src/test/java/io/stargate/web/docsapi/dao/DocumentDBTest.java

```diff
@@ -12,7 +12,7 @@
 import static org.mockito.Mockito.doNothing;
 import static org.mockito.Mockito.mock;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableList;
```

Comment:
not quite sure how this works now, but I think @tatu-at-datastax removed all direct guava usages and just depended on the shadowed version.. Would you be up using `Arrays.asList()` or `Collections.singeltonList()` just to avoid again the same old problem?